include/brod.hrl

```diff
@@ -17,104 +17,55 @@
 -ifndef(__BROD_HRL).
 -define(__BROD_HRL, true).
 
--type kafka_key()                 :: kpro:key().
```

Comment:
I'd like to keep all types in brod.hrl untouched at least for 2.x releases, but forward them to exported types. e.g. `kafka_value() :: brod:kafka_value().`
and `brod_client_id() :: brod:client_id().` % no need to repeat `brod` like `brod:brod_client_id()`